nanshe_workflow/par.py

```diff
@@ -138,7 +138,7 @@ def get_executor(client):
     try:
         yield executor
     finally:
-        executor.shutdown()
+        client.stop_dask()
```

Comment:
Seems we still need to call `executor.shutdown()` too. Fixing up in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/74 ).